hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java

```diff
@@ -260,7 +233,10 @@ private Event getApiEntrySpanForExitSpan(
 
     builder.setSpaceIds(EnrichedSpanUtils.getSpaceIds(event));
 
-    builder.setApiExitCalls(eventToApiExitCall.getOrDefault(event.getEventId(), 0));
+    builder.setApiExitCalls(
+        Integer.parseInt(
+            SpanAttributeUtils.getStringAttributeWithDefault(
```

Comment:
Don't we additional utility to get the integer attribute? Since, we are enriching an integer value, does it get enriched as a string?

Reply:
IIRC There isn't an utility which returns string parsed as Integer

Reply:
ah! I remember now too. We only support enriching string attributes. Got it